common/lib/xmodule/xmodule/tabs.py

```diff
@@ -375,6 +376,10 @@ def __init__(self, tab_dict=None):
         )
 
     def can_display(self, course, settings, is_user_authenticated, is_user_staff, is_user_enrolled):
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX', False):
+            from ccx.overrides import get_current_ccx
+            if get_current_ccx():
```

Comment:
I reviewed all the spots where we use the flag, and out of seven cases get_current_ccx is only used twice, so I don't think It's worth it.